src/gp/inc/GPMSAOptions.h

```diff
@@ -45,9 +46,6 @@ class GPMSAOptions
   //! Destructor
  ~GPMSAOptions();
```

Comment:
I should have made this `virtual` for readability.  It's not necessary, but I'll push that change directly to `dev`.